less/global.less

```diff
@@ -0,0 +1,40 @@
+// global styles
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  font-family: 'Roboto', 'Arial', sans-serif;
+  background: #ffffff;
+  color: #222222;
+}
+
+p {
+  line-height: 24px;
+  font-size: 1.6rem;
+}
+
+h2 {
+  font-size: 3rem;
+  padding-bottom: 10px;
+  font-weight: bold;
+}
+
+
+.container {
+  max-width:60%;
+  width: 100%;
```

Comment:
These two lines contradict each other. typically a max-width would be used with px. The way it is now, it will just always be 60% wide, because the max-width is less than the set width property.
Also, this 60% value seems too narrow for your content at the desktop screen view. Lots of whitespace around the sides and it's pushing your content too closely together. Maybe change this to something like 80%-85%?